mithril-aggregator/src/configuration.rs

```diff
@@ -153,6 +153,9 @@ pub struct Configuration {
     ///
     /// Will be ignored on (pre)production networks.
     pub allow_unparsable_block: bool,
+
+    /// Cardano transactions prover cache pool size
+    pub cardano_transactions_prover_cache_pool_size: u32,
```

Comment:
Why not define it with a `usize` type to avoid the conversion in the builder?

Reply:
Done